pkgs/applications/virtualization/lima/default.nix

```diff
@@ -8,16 +8,16 @@
 
 buildGoModule rec {
   pname = "lima";
-  version = "0.13.0";
+  version = "0.14.0";
```

Comment:
v0.14.1 now available

Reply:
It seems not to compile anyway just now: https://github.com/NixOS/nixpkgs/pull/205976/checks?check_run_id=10079994627

Reply:
And I have a feeling that it requires `codesign` in order to be able to use the new virtualization framework :(

Reply:
Opened https://github.com/NixOS/nixpkgs/pull/206285 for fixing all that.